packages/auth-providers/auth0/api/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@redwoodjs/auth-auth0-api",
+  "version": "3.2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redwoodjs/redwood.git",
+    "directory": "packages/auth-providers/auth0/api"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn build:js && yarn build:types",
+    "build:clean-dist": "rimraf 'dist/**/*/__tests__'",
+    "build:js": "babel src -d dist --extensions \".js,.ts,.tsx\" --copy-files --no-copy-ignored",
+    "build:types": "tsc --build --verbose",
+    "build:watch": "nodemon --watch src --ext \"js,ts,tsx,template\" --ignore dist --exec \"yarn build\"",
+    "prepublishOnly": "NODE_ENV=production yarn build",
+    "test": "jest src",
+    "test:watch": "yarn test --watch"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs3": "7.20.1",
+    "core-js": "3.26.0",
+    "jsonwebtoken": "8.5.1",
+    "uuid": "9.0.0"
+  },
+  "devDependencies": {
+    "@auth0/auth0-spa-js": "1.22.5",
+    "@babel/cli": "7.19.3",
+    "@babel/core": "7.20.2",
+    "@redwoodjs/auth": "3.2.0",
+    "@redwoodjs/cli-helpers": "3.2.0",
+    "@types/jsonwebtoken": "8.5.9",
+    "@types/react": "17.0.50",
+    "@types/uuid": "8.3.4",
```

Comment:
Don't need the types either (haven't looked at the rest of these yet)

```suggestion
```

Reply:
Done